### PR TITLE
support service postValidations part I - fix broken tests related to service postValidation

### PR DESCRIPTION
**Pull request by YannanGao-gs**
## Summary

Add graph support for service postValidation(https://github.com/finos/legend-engine/pull/1165), which will fix all current related broken tests.
## How did you test this change?

- [x] Test(s) added
- [ ] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

### CODE REVIEW

File: packages/legend-graph/src/graph/metamodel/pure/packageableElements/service/PostValidation.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import type { PostValidationAssertion } from './PostValidationAssertion.js';
+
+export class PostValidation implements Hashable {
+  description!: string;
+  parameters: RawLambda[] = [];
+  assertions: PostValidationAssertion[] = [];
+
+  get hashCode(): string {
+    return hashArray([
+      CORE_HASH_STRUCTURE.SERVICE_PARAMETER_VALUE,
```
Comment (gayathrir11) [inline packages/legend-graph/src/graph/metamodel/pure/packageableElements/service/PostValidation.ts:29]: Please change this to `CORE_HASH_STRUCTURE.SERVICE_POST_VALIDATION`

File: packages/legend-graph/src/graph/metamodel/pure/packageableElements/service/PostValidationAssertion.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import { CORE_HASH_STRUCTURE } from '../../../../../graph/Core_HashUtils.js';
+import type { RawLambda } from '../../rawValueSpecification/RawLambda.js';
+
+export class PostValidationAssertion implements Hashable {
+  id!: string;
+  assertion!: RawLambda;
+
+  get hashCode(): string {
+    return hashArray([
+      CORE_HASH_STRUCTURE.SERVICE_TEST,
```
Comment (gayathrir11) [inline packages/legend-graph/src/graph/metamodel/pure/packageableElements/service/PostValidationAssertion.ts:27]: Please change this to `CORE_HASH_STRUCTURE.SERVICE_POST_VALIDATION_ASSERTION`

File: packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureProtocol/serializationHelpers/V1_ServiceSerializationHelper.ts [hunk 3]
```diff
@@ -116,6 +118,28 @@ export const V1_parameterValueModelSchema = createModelSchema(
   },
 );
 
+const V1_servicePostValidationAssertionModelSchema = createModelSchema(
+  V1_PostValidationAssertion,
+  {
+    id: primitive(),
+    assertion: usingModelSchema(V1_rawLambdaModelSchema),
```
Comment (gayathrir11) [inline packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureProtocol/serializationHelpers/V1_ServiceSerializationHelper.ts:125]: Please arrange them in alphabetical order

File: packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureProtocol/serializationHelpers/V1_ServiceSerializationHelper.ts [hunk 4]
```diff
@@ -116,6 +118,28 @@ export const V1_parameterValueModelSchema = createModelSchema(
...
+);
+
+const V1_servicePostValidationModelSchema = createModelSchema(
+  V1_PostValidation,
+  {
+    description: primitive(),
+    parameters: list(usingModelSchema(V1_rawLambdaModelSchema)),
+    assertions: list(
+      custom(
+        (val) => serialize(V1_servicePostValidationAssertionModelSchema, val),
```
Comment (gayathrir11) [inline packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureProtocol/serializationHelpers/V1_ServiceSerializationHelper.ts:136]: same as above

**Comment by gayathrir11** [comment]
Can we add some tests?

File: packages/legend-graph/src/graph/metamodel/pure/packageableElements/service/PostValidation.ts (hunk 1 above)
Comment (YannanGao-gs) [reply packages/legend-graph/src/graph/metamodel/pure/packageableElements/service/PostValidation.ts:29]: opps! missed this! thanks!

File: packages/legend-graph/src/graph/metamodel/pure/packageableElements/service/PostValidationAssertion.ts (hunk 2 above)
Comment (YannanGao-gs) [reply packages/legend-graph/src/graph/metamodel/pure/packageableElements/service/PostValidationAssertion.ts:27]: thanks 💟

File: packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureProtocol/serializationHelpers/V1_ServiceSerializationHelper.ts (hunk 3 above)
Comment (YannanGao-gs) [reply packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureProtocol/serializationHelpers/V1_ServiceSerializationHelper.ts:124]: > Can we add some tests?

one roundtrip test was added. thanks!

**Comment by YannanGao-gs** [comment]
> Can we add some tests?

one roundtrip test was added. :)

**Review by MauricioUyaguari — APPROVED** [review]

**Pull request merged**